wallet/public/index.html

```diff
@@ -31,93 +53,22 @@
-
       Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Agoric Wallet</title>
-    <script type="module">
-      const MAX_ATTEMPTS = 10;
-      const WAIT_PER_ATTEMPT = 10;
-      let attempts = 0;
-      const startWhenKeplrAvailable = () => {
-        console.debug('looking for keplr');
-        attempts += 1;
-        if (window.keplr) {
-          // XXX copied from lockdown.ts; could be automated
-          // Allow the React dev environment to extend the console for debugging
-          // features.
-          // @ts-expect-error template
-          // eslint-disable-next-line no-constant-condition
-          const consoleTaming =
-            '%NODE_ENV%' === 'development' ? 'unsafe' : 'safe';
-          // @ts-expect-error template
-          // eslint-disable-next-line no-constant-condition
-          const errorTaming =
-            '%NODE_ENV%' === 'development' ? 'unsafe' : 'safe';
-
-          // eslint-disable-next-line no-restricted-properties
-          const { pow: mathPow } = Math;
-          // eslint-disable-next-line no-restricted-properties
-          Math.pow = (base, exp) =>
-            typeof base === 'bigint' && typeof exp === 'bigint'
-              ? base ** exp
-              : mathPow(base, exp);
-
-          lockdown({
-            errorTaming,
-            overrideTaming: 'severe',
-            consoleTaming,
-          });
-
-          console.log('lockdown done.');
-
-          window.addEventListener('unhandledrejection', ev => {
-            ev.stopImmediatePropagation();
-          });
-
-          console.log('Importing app bundle');
-          // iterate manually to load by dynamic import
-          <% htmlWebpackPlugin.files.js.forEach(function(jsPath) { %>
-            import('<%= jsPath %>');
-          <% }); %>
-        } else {
-          if (attempts < MAX_ATTEMPTS) {
-            setTimeout(startWhenKeplrAvailable, WAIT_PER_ATTEMPT);
-          } else {
-            const notice = document.getElementById('keplr-missing');
-            notice.hidden = false;
-          }
-        }
-      };
-      startWhenKeplrAvailable();
-    </script>
-    <!--
-      this output is normally produced by htmlWebpackPlugin.tags.headTags or
-      inject:true but we need to defer the script tags so do this part manually
-    -->
-    <% htmlWebpackPlugin.files.css.forEach(function(cssPath) { %>
-    <link href="<%= cssPath %>" rel="stylesheet" />
-    <% }); %>
   </head>
-
   <body style="overflow-x: hidden; overflow-y: scroll">
-    <%= htmlWebpackPlugin.tags.bodyTags %>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root">
-      <!-- replaced by React mounting -->
-    </div>
-    <div id="keplr-missing" hidden>
```

Comment:
what will happen when the Keplr extension isn't installed?

Reply:
It will error, same behavior as before